functional-tests/mining_sync_test.go

```diff
@@ -84,6 +84,10 @@ func TestBootstrapWindowedPoSt(t *testing.T) {
 	wd, _ := os.Getwd()
 	genCfgPath := filepath.Join(wd, "..", "fixtures/setup.json")
 	presealPath := filepath.Join(wd, "..", "fixtures/genesis-sectors")
+	// setup presealed sectors and uncomment to run test against sectors with larger sector size
+	//genCfgPath := filepath.Join("./512", "setup.json")
+	//presealPath := "./512"
```

Comment:
This is how I ran the 512MiB test. It's probably too much data for a fixture, and makes for longer functional tests (although still less than a minute on my laptop).

Reply:
fancy laptop